### PR TITLE
Update Readme With Example

### PR DESCRIPTION
**Pull request by yahilmadakiya**
This PR adds example of `Inspector Controls`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,9 +65,35 @@ register_block_type( 'example-namespace/example-block', array(
 			)
 		)
 	),
+	'render_callback' => 'example_callback',
 ) );
 ```
 
+**Inspector Controls:**
+
+Inspector controls are shown in the sidebar when a block is selected and can easily be added by using `placement: 'inspector'` when adding attributes as shown in the example below.
```
Comment (danielbachhuber) [inline README.md]: Can you include this within the original JS example?

Comment (yahilmadakiya) [reply README.md]: @danielbachhuber  Please confirm.

You mean add the inspector control example here in the first JS example?
```js
registerBlockType( 'example-namespace/example-block', {
    title: 'Example Block',
    url: {
        type: 'string',
        field: {
            type: 'url',
        },
    },
    text: {
        type: 'string',
        field: {
            type: 'text',
            placeholder: 'Enter link text',
        },
    },
});
```

Comment (danielbachhuber) [reply README.md]: Yes, add it to the existing example instead of creating a new one. It's more obvious to the end user.

Comment (danielbachhuber) [reply README.md]: Good, although this comment probably no longer applies:

image

Can you remove, or incorporate it as a code comment in the JS example?

Comment (yahilmadakiya) [reply README.md]: Added as a code comment.

**Pull request merged**